src/get_data_completed_project_package/project_data_serializer.py

```diff
@@ -0,0 +1,21 @@
+class ProjectDataSerializer:
```

Comment:
В названии `Serializer`, что указывает на то что он должен что-то делать, но по факту класс просто ДТО без методов

```suggestion
class SerializedProjectData:
```

Reply:
Согласен, может тогда Serialize убрать полностью и вместо него добавить DTO в имени?